people-and-planet-ai/geospatial-classification/README.ipynb

```diff
@@ -0,0 +1,1459 @@
+   },
+   "source": [
+    "## ✨ Before you begin, you need to…\n",
+    "\n",
+    "1. Decide on creating a new\n",
+    "   [free project](https://cloud.google.com/free/docs/gcp-free-tier)\n",
+    "   _(recommended)_ or using an existing one.\n",
+    "   Then **copy the project ID** and paste it in the `google_cloud_project` field in the \"Entering project details” section below.\n",
+    "\n",
+    "   > 💡 If you _don't plan to keep the resources_ that you create via this sample, we recommend creating a new project instead of selecting an existing project.\n",
+    "   > After you finish these steps, you can delete the project, removing all the resources associated in bulk.\n",
+    "\n",
+    "1. Make sure that **billing is enabled** for your Google Cloud project,\n",
+    "   [_click here_](https://cloud.google.com/billing/docs/how-to/modify-project)\n",
+    "   to learn how to confirm that billing is enabled.\n",
+    "\n",
+    "1. Have an **Earth Engine** account (it's FREE) or create a new one.\n",
+    "  To create an account, fill out the [registration form here.](https://signup.earthengine.google.com/#!/). Please note this can take from 0-24 hours...but it's worth it! Come back to this sample after you have this."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "156ab72c",
+   "metadata": {
+    "id": "156ab72c"
+   },
+   "source": [
+    "### ⛏️ Preparing the project environment\n",
+    "\n",
+    "Click the run ▶️ icons in order for the cells to download and install the necessary code, libraries, and resources for this solution.\n",
+    "\n",
+    "> 💡 You can _optionally_ view the entire\n",
+    "> [code in GitHub](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/people-and-planet-ai/geospatial-classification)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa34fdd8",
+   "metadata": {
+    "id": "fa34fdd8"
+   },
+   "source": [
+    "### ↘️ Get the code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c2848f6d",
+   "metadata": {
+    "id": "c2848f6d"
+   },
+   "outputs": [],
+   "source": [
+    "# Get the sample source code.\n",
+    "\n",
+    "!git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git ~/python-docs-samples\n",
+    "%cd ~/python-docs-samples/people-and-planet-ai/geospatial-classification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "bb8c6ede",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 1000
+    },
+    "id": "bb8c6ede",
+    "outputId": "3079b40a-9d74-4441-db83-bf6c7bcf58e2"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -r requirements.txt -c constraints.txt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b86fba8",
+   "metadata": {
+    "id": "3b86fba8"
+   },
+   "outputs": [],
+   "source": [
+    "%cd ~/python-docs-samples/people-and-planet-ai/geospatial-classification"
```

Comment:
Nit: sorry for the nitpicking, but can we move this to the end of the next cell? That way whenever we authenticate we make sure we're in the right directory